templates/addons/calico.yaml

```diff
@@ -11,12 +11,11 @@ data:
   typha_service_name: "none"
   # Configure the backend to use.
   calico_backend: "vxlan"
+
   # Configure the MTU to use for workload interfaces and tunnels.
-  # - If Wireguard is enabled, set to your network MTU - 60
-  # - Otherwise, if VXLAN or BPF mode is enabled, set to your network MTU - 50
-  # - Otherwise, if IPIP is enabled, set to your network MTU - 20
-  # - Otherwise, if not using any encapsulation, set to your network MTU.
-  veth_mtu: "1440"
+  # By default, MTU is auto-detected, and explicitly setting this field should not be required.
+  # You can override auto-detection by providing a non-zero value.
+  veth_mtu: "0"
```

Comment:
@jsturtevant I'm thinking this might fix #1353 ? Took this from the vxlan manifest https://docs.projectcalico.org/v3.19/manifests/calico-vxlan.yaml

Reply:
I've reverted this and all other template changes except for the version bump, something is causing Calico to crash (see #1584)